projects/eods-specification.md

```diff
@@ -0,0 +1,266 @@
+
+  Ethereum Protocol's credibility comes from the control over the Validators that execute the protocol. But it can only control what it can see, so it's important to extend these limits, in order to allow for the Protocol to have the capacity to react with automated defense systems.
+
+- Currently, ETH delegators do not play a [meaningful role](https://epf.wiki/#/wiki/research/eODS?id=the-role-of-delegators) in the protocol, as they don't actively participate in Consensus. We can improve this by allowing the Protocol to identify delegated stake and incentivize that role selection. Delegators under eODS model do not contribute to the economic security of FFG, i.e. Delegators do not partake in Finality (non-slashable stake), but they are able to surface discrepancies in the gadget’s functioning. Their services can be compensated by re-allocated aggregated issuance.
+    
+    Delegator role, under Operator-Delegator separation:
+    * The curation of operator set: Opinionated delegators may decide to choose between different operators based on e.g., fees or reliability. These criteria could be part of a Validator rating system developed either on CL clients side or in-protocol.
+    * The provision of non-FFG services: The delegators may be called upon to provide non-slashable yet critical services, like:
+        * input their view into censorship-resistance gadgets such as inclusion lists or multiplicity gadgets.
+        * co-sign block proposals, attestations together with the bonded Gasper operators.
+ 
+    This is important, so we can move away from the unincentivized delegate selection practiced today in delegated proof of stake, which is basically [a flawed type of voting](https://notes.ethereum.org/@vbuterin/staking_2023_10#Expanding-delegate-selection-powers).
+  
+- Building towards the inevitable implementation of SSF requires taking the best possible trade-offs.
+Identifying, disambiguating, or solving these trade-offs is not trivial, as they depend on a wide range of challenges and limitations:
+    - physical network limitations (e.g. computing power) 
+    - technical limitations (e.g. per slot BLS signatures aggregation)
+    - not-just-technical fundamentals (e.g. community goals and values vs. out-of-protocol market forces)
+
+  This is important in our journey to build Ethereum as the envisioned global-scale network, because in that scenario, single-slot finality is not only desirable but most likely mandatory, and I feel that if we're building towards the right goals, we better be building towards SSF. Regardless what the goals set by the Ethereum community turn out to be, enshrining ODS would help the community impose those goals, by exercising better control over the validators that execute the protocol, while addressing the challenges mentioned above.
```

Comment:
This too long and repetitive for the project motivation, it would rather fit specs or description. No need to edit it now, just for future reference, no need to elaborate too much